### PR TITLE
fix: [FFM-3552]: Updating + from string to icon, adjusting padding on New Flag Button

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

I have updated the + in the 'New Flag' button to be an icon where as previously it was just a character. I have also adjusted the layout of the button according to the design spec—no need to change the three-dot icon anymore as this has been addressed.

#### Screenshots

image
New

image
Live

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by crystal-fong — APPROVED** [review]
theyre so consistent now!

File: src/modules/75-cf/components/CreateFlagButton/CreateFlagButton.module.scss [hunk 1]
```diff
@@ -6,5 +6,5 @@
  */
 
 .open-modal-btn {
-  padding: var(--spacing-small) var(--spacing-huge) !important;
+  padding: var(--spacing-small) var(--spacing-medium) var(--spacing-small) 12px !important;
```
Comment (knagurski) [inline src/modules/75-cf/components/CreateFlagButton/CreateFlagButton.module.scss:9]: Ideally we shouldn't be using any custom CSS for a button. Can we see what this looks like with the custom CSS removed altogether? Having special cases for individual buttons means that if the underlying button style changes we then have to fix this.

Comment (laura-mannus) [reply src/modules/75-cf/components/CreateFlagButton/CreateFlagButton.module.scss]: Thank you, I have deleted this file

**Comment by laura-mannus** [comment]
retrigger eslint

**Review by crystal-fong — APPROVED** [review]

**Review by knagurski — APPROVED** [review]
🔥

**Pull request merged**